pkg/fleet-manager/application/rollout_helper.go

```diff
@@ -167,6 +167,48 @@ func (a *ApplicationManager) syncRolloutPolicyForCluster(ctx context.Context,
 	return ctrl.Result{}, nil
 }
 
+func (a *ApplicationManager) deleteResourcesInClusters(ctx context.Context, app *applicationapi.Application, fleet *fleetapi.Fleet) error {
```

Comment:
deleteResourcesInClusters is to remove resources in member cluster?

If so i would rename it to `deleteResourcesInMemberClusters`